web/src/server/api/routers/scores.ts

```diff
@@ -322,6 +328,7 @@ export const scoresRouter = createTRPCRouter({
         },
       });
 
+      // TODO: Can we drop this upsert logic and just error if a score already exists?
```

Comment:
@maxdeichmann I think we should drop the `upsert` logic within the create endpoint here. It doesn't seem to be used (only reference in https://github.com/langfuse/langfuse/blob/main/web/src/features/scores/components/AnnotateDrawerContent.tsx#L398) and may produce unintuitive results in my view. 

I can check whether a record already exists for the given Clickhouse key or just create an insert that overwrites whatever is there. The former is probably preferable, but may increase load.

Let me know if you see any reason to keep this bit around and supported, otherwise, I'll go ahead with the removal tomorrow. I also checked with @marliessophie via Slack, but didn't get her assessment yet.

Reply:
I would overwrite any existing score in clickhouse. With whatever should be updated. Hence get from CH and replace. @marliessophie what is your take?